CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changes
 4.4.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Add support for general-purpose Python 3.x annotations / type-hints in interface definitions.
```

Comment:
I wonder if that may be overstating things a bit. As I understand it, this simply drops top-level `__annotations__` completely from the class at runtime. It allows Attribute values to thus be annotated in the source, but that's about it. For something that evaluates the AST (mypy?) that's probably OK, but for something that uses that data at runtime, like the stdlib `typing` module, that's not exactly "supported". (Indeed, `tpying.get_type_hints` raises an exception when passed an interface: `TypeError: <InterfaceClass __main__.IFoo> is not a module, class, method, or function.`)

In addition, methods also still completely lose their annotations (because they're represented by a custom type). E.g., this works with the released versions of zope.interface today:
```python
class IFoo(Interface):
   def thing(s:str) -> str:
       pass
```

But `IFoo['thing']` has no `__annotations__`.

Was your use-case mypy or something else?